torchvision/transforms/functional_tensor.py

```diff
@@ -81,13 +90,14 @@ def adjust_brightness(img, brightness_factor):
     Returns:
         Tensor: Brightness adjusted image.
     """
-    if not F._is_tensor_image(img):
+    if not len(img.shape) == 3:
         raise TypeError('tensor is not a torch image.')
 
-    return _blend(img, 0, brightness_factor)
+    return _blend(img, torch.zeros(img.shape), brightness_factor)
```

Comment:
Can you make this `torch.zeros_like(img)`, so that it works with `uint8` and other types as well?